ENIGMAsystem/SHELL/Graphics_Systems/OpenGLES3/std.cpp

```diff
@@ -6,4 +6,8 @@ const gl_profile_type graphics_opengl_profile = gl_profile_es;
 const unsigned int gl_major = 3, gl_minor = 2; 
 const std::string shader_header = "#version 320 es\nprecision mediump float;\n";
 
+GLenum shadertypes[5] = {
```

Comment:
Also, why not throw this into GL-Desktop, since it's used in both desktop systems?

Reply:
cause gl1 prolly not support all idk?